api/scpca_portal/metadata_file.py

```diff
@@ -76,6 +76,14 @@ def load_library_metadata(metadata_file_path: Path):
         return transform_keys(json.load(raw_file), LIBRARY_METADATA_KEYS)
 
 
+def format_metadata_dict(metadata_dict: Dict) -> Dict:
+    """
+    Returns a copy of transformed dict after converting its value to a joined string from the list.
+    Otherwise returns metadata_dict.
```

Comment:
```suggestion
    Returns a copy of metadata dict that is formatted and ready to be written to file.
```

Just for clarity here, the actual changes don't matter as those are in the doc strings of the util functions. We should be clear here that the dict is always copied.

Reply:
See commit at 8cdcd9b553b62e86ee21c1df80a6f2a43b2c120a.